### ISSUE TITLE
Does stopping the model early in Google Colab lose training progress?

### ISSUE THREAD

**Issue by rbb091020** [alankbi/detecto#26]
I've been training with model.fit() for about 5 horus now on a dataset of about ~1200 images. I see now that I should have converted the xml's into a single csv file to speed up data transfer.  If I were to stop the model in a Google Colab notebook, would I lose the training progress?

**Comment by alankbi** [comment]
This was answered over email, but I'm pasting the answer here in case others have the same question:

If you press the stop button, my intuition is that the model would be partially trained, but I have not tested that before so I can't say for sure. 

To tell how far along you are while training, you should also set `verbose=True` when calling the fit method, as this will tell you what epoch you're on and whether the loss is decreasing (if you provided a validation dataset). This is helpful as if your loss is no longer decreasing, then that may be an indication that your model has reached close to peak performance and you don't need to train for any more epochs. 

Lastly, please check that the GPU is enabled on your Colab notebook (Edit -> Notebook settings) as 5 hours is a long time! Also note that Colab times out after 30 minutes of inactivity or 12 hours so you don't want to lose your state. 

P.S. that part in the docs about CSV files saving time is only partly true; it can help speed it up a tiny bit but should not make a major difference in the overall training time. I've updated the docs to make it a bit clearer.
